From: Sune Vuorela <sune@debian.org>
Subject: Make kpythonfactory dlopen versioned python library.
Bug-Debian: http://bugs.debian.org/524685
Bug-Debian: http://bugs.debian.org/541999
Last-Update: 2009-09-19
Forwarded: no
Origin: vendor

Otherwise, python-dev would be needed for Python Plugin factory to work.

--- kpythonpluginfactory/kpythonpluginfactory.cpp.orig
+++ kpythonpluginfactory/kpythonpluginfactory.cpp
@@ -307,7 +307,7 @@ QLibrary *LoadPythonLibrary()
 {
     QLibrary *pythonLib = new QLibrary();
     pythonLib->setLoadHints(QLibrary::ExportExternalSymbolsHint);
-    pythonLib->setFileName(LIB_PYTHON);
+    pythonLib->setFileName(LIB_PYTHON ".1");
     pythonLib->load();
     return pythonLib;
 }
